DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Factory/AnonymizerSettingsFactory.cs

```diff
@@ -0,0 +1,45 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.Exceptions;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors.Settings
+{
+    public class AnonymizerSettingsFactory : IAnonymizerSettingsFactory
```

Comment:
>AnonymizerSettingsFactory [](http://example.com/codeflow?start=17&length=25)

looks like you don't need to create an instance for this. How about changing this into a static class with the function CreateAnonymizerSetting?